apix-impl/src/integration-test/java/eu/xenit/apix/rest/v1/tests/CopyNodeTest.java

```diff
@@ -52,19 +61,49 @@ public void testCopyNode() {
         final ChildParentAssociation primaryParentAssoc = (ChildParentAssociation) parentAssociations.get(0);
         final NodeRef parentRef = primaryParentAssoc.getTarget();
 
-        final String url = makeAlfrescoBaseurl("admin", "admin") + "/apix/v1/nodes";
+        final String url = makeAlfrescoBaseurl("admin", "admin") + "/apix/v2/nodes";
```

Comment:
@Ranecdev The test now points to v2 of the REST API but these are a test set for the v1 REST API.
Changes have been made to v1 and v2 webscripts. Test should be provided for both v1 and v2 REST endpoints
Tests on v1 should also contain requests with the previous set of parameters since we also want to test we didn't brake the API

Reply:
> Tests on v1 should also contain requests with the previous set of parameters since we also want to test we didn't brake the API

Should this new behavior only be added to v2? (Also their are some olders tests that are still functioning)

Reply:
I believe this change is a fix, so that the endpoint works as intended. I think it should be fixed in both versions.
I just want to make sure that a previous request format to the v1 endpoint still keeps working